crud/src/main/java/com/redhat/lightblue/rest/crud/health/CrudCheckRegistry.java

```diff
@@ -17,15 +17,15 @@ private CrudCheckRegistry() {
 
     public static HealthCheckRegistry getDiagnosticCheckRegistry() {
         if(diagnosticCheckRegistry == null) {
-            diagnosticCheckRegistry = CrudCheckConfiguration.getDiagnosticCheckRegistry();
+            diagnosticCheckRegistry = CrudCheckConfiguration.createDiagnosticCheckRegistry();
         }
         return diagnosticCheckRegistry;
     }
 
 
     public static HealthCheckRegistry getHealthCheckRegistry() {
         if(healthCheckRegistry == null) {
-            healthCheckRegistry = CrudCheckConfiguration.getHealthCheckRegistry();
+            healthCheckRegistry = CrudCheckConfiguration.createHealthCheckRegistry();
         }
         return healthCheckRegistry;
     }
```

Comment:
Do we still need both CrudCheckRegistry and CrudCheckConfiguration?

Reply:
Not really, merged everything into CrudCheckRegistry